### DISCUSSION TITLE
Frient/Develco Meter interface ZHEMI101 - Kamstrup / IEC62056-21 mode

### DISCUSSION THREAD

**Discussion by cloverleyhall** [Koenkk/zigbee2mqtt#15018] [Devices]
Hi,
I've got a few of [these](https://www.zigbee2mqtt.io/devices/ZHEMI101.html) units to measure energy consumption on a district heating network.

I managed to get hold of the IR probes in order to read the heat meters (Kamstrup Multical 402) through the optical port.

The devices connect to Z2M fine but always report 4295032.83 kWh

Before I go down the route of submitting this as an "issue", I just wanted to find out if anyone else out there has successfully used these units in "Kamstrup-kmp" mode?

Or even "IEC62056-21" mode? The same thing happens when I connect to our electricity meter.

**Comment by Medzez** [comment]
Can i ask where you managed to get hold of an IR probe?

**Comment by cloverleyhall** [comment]
I emailed Develco and asked very politely if they would sell me some. Ultimately I could never get it to work so eventually I cut the mini jack plugs off and wired them to an ESP device so now I'm reading the meters with ESPhome.

I'm pretty sure you could get something like this: [https://www.aliexpress.com/item/1005004918762123.html](https://www.aliexpress.com/item/1005004918762123.html)
and put a TRRS mini jack plug on the end to plug into the ZHEMI101
